docs/api-overview.md

```diff
@@ -24,9 +24,28 @@
 
 # API documentation
 
-The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM.
+The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM. The documentation also includes links to the API documentation for the Java&trade; SE and JDK reference implementation.
+
+
+## Performance
+
+If your Java application manipulates native data, the Data Access Accelerator API package (`com.ibm.dataacess`) can help improve application performance. Classes are available for manipulating packed decimal data, converting between decimal data types, and marshalling Java binary types to and from byte arrays.

```

Comment:
@fjeremic asking you as I think DAA came out of the z-ecosystem - what's the primary use case for it?  Financial services?  Cobol interop?  Something else?

Hopefully someone with more DAA experience can help position this more clearly